ocaml/xcp-rrdd/lib/plugin/reporter.ml

```diff
@@ -57,10 +59,20 @@ type state =
   | Cancelled
   | Stopped of [`New | `Cancelled | `Failed of exn]
```

Comment:
Unrelated - but I would have preferred this to be a regular sum type. This module needs to be in scope so I don't see the benefit to use a polymorphic sum type.